tools/lib/capitalization.py

```diff
@@ -24,6 +24,7 @@
     r"ID",
     r"IDs",
     r"IP",
+    r"Inbox",
```

Comment:
You shouldn't need to be adding these as linter exceptions; (1) they follow our linting rules, and (2) I don't think it works well for translators to have this as a variable passed around.

Reply:
`tools/check-capitalization` gives this error
```
Strings not properly capitalized in frontend:
The <b>Inbox</b> view provides an overview of your conversations with unread messages.
```

Reply:
Hm, maybe it's best to follow our own rules and not capitalize those. :)

Reply:
We decided to use capitalization in https://github.com/zulip/zulip/pull/29192#discussion_r1520565943